docs/UserGuide.adoc

```diff
@@ -168,6 +168,24 @@ Deletes the 2nd person in the address book.
 `delete 1` +
 Deletes the 1st person in the results of the `find` command.
 
+==== To delete multiple persons:
+
+Deletes the specified person from the address book. +
+Format: `deletem INDEX...`
+
```

Comment:
just wondering if we can extend the existing delete function to take multiple indexes instead. this would allow us to simply use `delete` as the command which seems more user friendly